src/app.js

```diff
@@ -1,38 +1,38 @@
 /**
  * Created by championswimmer on 28/5/15.
  */
-'use strict';
+"use strict";
 // Declare app level module which depends on views, and components
 
-var openevent = angular.module('openevent',
+var openevent = angular.module("openevent",
     [
-        'ui.router',
-        'ngStorage',
-        'ngMaterial',
-        'leaflet-directive',
-        'oe.sidenav',
-        'oe.sessions',
-        'oe.speakers',
-        'oe.tracks',
-        'oe.sponsors',
-        'oe.twitterwall',
-        'oe.map'
+        "ui.router",
+        "ngStorage",
+        "ngMaterial",
+        "leaflet-directive",
+        "oe.sidenav",
+        "oe.sessions",
+        "oe.speakers",
+        "oe.tracks",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2411866147/issues/source?bid=3233857&fileBranchId=3235518#l16)